client/rest/src/test/java/org/elasticsearch/client/documentation/RestClientDocumentation.java

```diff
@@ -91,7 +91,7 @@
     // end::rest-client-options-singleton
 
     @SuppressWarnings("unused")
-    public void testUsage() throws IOException, InterruptedException {
+    public void usage() throws IOException, InterruptedException {
```

Comment:
These are for documentation only, not tests, it's not code we run and the method name is not important.
I changed it because testing conventions picked up the method name.

Reply:
I *thought* we did execute these. In my opinion, it is important to execute the examples that we have in the docs. They may not assert all that much, but at a minimum they shouldn't crash and it'd be nice if we made them assert things that matched the prose. Which we often do and might do here as well.